packages/astro/test/fixtures/astro-basic/src/pages/fileurl.astro

```diff
@@ -0,0 +1,10 @@
+---
+import {capitalize} from 'file://../strings.js';
```

Comment:
Is this supposed to be a thing you can do? I've always assumed `file:` required an absolute path

Reply:
This is how npm suggests doing it: https://docs.npmjs.com/cli/v10/configuring-npm/package-json although there they are omitting the `//` part, either works, try `new URL('file://../strings.js')` in your console.

Although in practice this feature is for full URLs passed via `new URL('./rel', import.meta.url)`, I think this added bonus is kind of cool, but it was really only done this way because it's easier to test.

Reply:
Hmm, TIL! Thanks for clarifying